tutorials/mpi-introduction/zh_cn.md

```diff
@@ -18,7 +18,7 @@ redirect_from: '/mpi-introduction/zh_cn'
 ## MPI 的历史简介
 在 90 年代之前，程序员可没我们这么幸运。对于不同的计算架构写并发程序是一件困难而且冗长的事情。当时，很多软件库可以帮助写并发程序，但是没有一个大家都接受的标准来做这个事情。
 
-在当时，大多数的并发程序只出现在科学和研究的领域。最广为接受的模型就是消息传递模型。什么是消息传递模型？它其实只是指程序通过在进程间传递消息（消息可以理解成带有一些信息和数据的一个数据结构）来完成某些任务。在实践中，并发程序用这个模型去实现特别容易。举例来说，主进程（master process）可以通过对从进程（slave process）发送一个描述工作的消息来把这个工作分配给它。另一个例子就是一个并发的排序程序可以在当前进程中对当前进程可见的（我们称作本地的，locally）数据进行排序，然后把排好序的数据发送的邻居进程上面来进行合并的操作。几乎所有的并行程序可以使用消息传递模型来描述。
```

Comment:
@cckn1ght - Would you be able to confirm that the terminology in the translation here is still acceptable after replacing these words?

Reply:
Nevermind, I was able to get a quick clarification from someone on my team who could help out. They confirmed that the translation is still fine.